recipes/khiopsdriver-gcs/conda_build_config.yaml

```diff
@@ -0,0 +1,13 @@
+---
+cxx_compiler:
+  - gxx # [linux]
+  - clangxx # [osx]
+  - vs2022 # [win]
+cxx_compiler_version:
+  - 12 # [linux]
+  - 16 # [osx]
```

Comment:
Does this package really need older versions of the compilers?  If not, then please remove these lines.

Reply:
Removed.